src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala

```diff
@@ -106,6 +109,15 @@ object SchemaTools extends SparkSessionWrapper {
    * @return
    */
   def cullNestedColumns(df: DataFrame, structToModify: String, nestedFieldsToCull: Array[String]): DataFrame = {
+    //Exception Block
+    if (nestedFieldsToCull.filter(_.contains(".")).size != 0) {
```

Comment:
switch to 
`nestedFieldsToCull.exists(_.contains("."))`

Reply:
Modified